lua/entities/cfc_trampoline/cl_init.lua

```diff
@@ -3,3 +3,11 @@ include( "shared.lua" )
 function ENT:Draw()
     self:DrawModel()
 end
+
+local BOUNCE_SFX_VOLUME = CreateClientConVar( "cfc_trampoline_volume", 1, true, false, "Volume for the trampoline", 0, 1 )
+
+net.Receive( "CFC_BouncyCircle_PlayBounceSound", function()
+    local ent = net.ReadEntity()
+
+    ent:EmitSound( "cfc/cfc_trampoline/cfc_trampoline_bounce.ogg", 75, 100, BOUNCE_SFX_VOLUME:GetFloat() )

```

Comment:
We may need an `IsValid` check here. I think the item could be in your PVS but not your PAS? Not 100% sure.
Let's merge it without and then see if clients error.

Also, a specific [CHAN](https://gmodwiki.com/Enums/CHAN) might be good here but I don't know which one. It's probably fine without.